src/main/java/com/taxi/util/ConnectionUtil.java

```diff
@@ -0,0 +1,38 @@
+package com.taxi.util;
+
+import com.taxi.exception.DataProcessingException;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ConnectionUtil {
+    @Value("datasource.url")
+    private String url;
+    @Value("datasource.username")
+    private String name;
+    @Value("datasource.password")
+    private String password;
+
+    public Connection getConnection() {
+        Connection connection = null;
+        Properties properties = null;
+
+        try {
+            properties = new Properties();
+            properties.load(new FileInputStream("src/main/resources/application.properties"));
+            connection = DriverManager.getConnection(properties.getProperty(url),
+                    properties.getProperty(name),
+                    properties.getProperty(password));
```

Comment:
```suggestion
            connection = DriverManager.getConnection(url, name, password);
```
Идея состоит в том чтобы отказаться от считывания файла вручную)